db/cache/charts_test.go

```diff
@@ -45,7 +45,7 @@ func TestMain(m *testing.M) {
 func TestChartsCache(t *testing.T) {
 	gobPath := filepath.Join(tempDir, "log.gob")
 	ctx, shutdown := context.WithCancel(context.Background())
-	charts := NewChartData(0, time.Unix(0, 0), &chaincfg.MainNetParams, ctx)
+	charts := NewChartData(ctx, 0, time.Unix(0, 0), &chaincfg.MainNetParams)
```

Comment:
@buck54321 Having context as the first arg is good practice.  Might as well make the same change in https://github.com/decred/dcrdata/pull/1384 too if you decide to modify the signature of `NewChartData`